modules/generator/processor/spanmetrics/spanmetrics.go

```diff
@@ -84,21 +109,28 @@ func (p *Processor) Shutdown(_ context.Context) {
 
 func (p *Processor) aggregateMetrics(resourceSpans []*v1_trace.ResourceSpans) {
 	for _, rs := range resourceSpans {
-		// already extract service name, so we only have to do it once per batch of spans
+		// already extract job name & instance id, so we only have to do it once per batch of spans
 		svcName, _ := processor_util.FindServiceName(rs.Resource.Attributes)
-
+		jobName := processor_util.GetJobValue(rs.Resource.Attributes)
+		instanceID, _ := processor_util.FindInstanceID(rs.Resource.Attributes)
+		if p.Cfg.EnableTargetInfo {
+			resourceLabels := processor_util.GetTargetInfoAttributes(rs.Resource.Attributes)
+			p.targetInfoLabels = append(p.targetInfoLabels, resourceLabels...)
```

Comment:
are we adding `service.name`, `service.namespace`, and `service.instance.id` as target_info labels?